numpy/lib/npyio.py

```diff
@@ -1119,6 +1113,17 @@ def read_data(chunk_size):
             warnings.warn('loadtxt: Empty input file: "%s"' % fname,
                           stacklevel=2)
 
+        line_iter = itertools.islice(
+            itertools.chain([first_line], fh), max_rows)
+        if isinstance(first_line, bytes):
+            decoder = methodcaller(
+                "decode", encoding if encoding is not None else "latin1")
+            line_iter = map(decoder, line_iter)
+
+        lineno_words_iter = filter(
+            itemgetter(1),  # item[1] is words; filter skips empty lines.
+            enumerate(map(split_line, line_iter), 1 + skiprows))
```

Comment:
This `1 +` looks like a behavior change to me - didn't the line numbers start at 0 before?